src/frontend/screens/ObservationEdit/LocationView.tsx

```diff
@@ -19,66 +19,95 @@ const m = defineMessages({
 });
 
 export const LocationView = () => {
-  const liveLocation = useMostAccurateLocationForObservation();
-  const observationValue = usePersistedDraftObservation(
-    observationValueSelector,
+  const observationValue = usePersistedDraftObservation(store => store.value);
+
+  return observationValue?.metadata.manualLocation ? (
+    <LocationViewManualPosition />
+  ) : (
+    <LocationViewUpdatePosition />
+  );
+};
+
+const LocationViewManualPosition = () => {
+  const observationValue = usePersistedDraftObservation(store => store.value);
+  return (
+    <LocationViewInner
+      lat={observationValue?.lat}
+      lon={observationValue?.lon}
+      accuracy={null}
+    />
   );
-  const coordinateFormat = usePersistedSettings(coordinateFormatSelector);
+};
+
+const LocationViewUpdatePosition = () => {
+  const observationValue = usePersistedDraftObservation(store => store.value);
+  //only update the location if the accuracy increases OR the user has moved outside of the accuracy's radius of uncertainty.
+  const {location} = useLocation(
+    ({accuracy}) => accuracy === 'better' || accuracy === 'newBounds',
+  );
+  const {updateObservationPosition} = useDraftObservation();
+
+  useEffect(() => {
+    const newCoord = !location
+      ? undefined
+      : Object.entries(location.coords).map(
```

Comment:
Why are we converting the coordinates into entries and then back again?

Reply:
it because expo location returns `{value} | null` and our backend expects `{value} | undefined`

Reply:
https://github.com/digidem/comapeo-mobile/pull/387/commits/e3d3d89dd25f39826a29ad249b5bb9b6481388a2